.github/linters/.arm-ttk.psd1

```diff
@@ -9,5 +9,6 @@
         'DependsOn Best Practices',
         'Outputs Must Not Contain Secrets',
         'IDs Should Be Derived From ResourceIDs'
```

Comment:
should there be a comma at the end? if not, pls ignore / close this comment. 
thanks

Reply:
You are right, it is optional, but I would like to set a comma for consistency reasons in other locations.